oat-sa/extension-tao-itemqti-pci
# Feature/TR-765/load test taker response in audio PCI

Related to: https://oat-sa.atlassian.net/browse/TR-765

Make sure the `audioRecording` PCI can accept a serialized state or a response as a state.

When reviewing the response, the state is restored from a serialized version, and the object contains an additional level.

To apply the changes, you can either run `taoUpdate` or directly the migration script:
```
php index.php '\oat\tao\scripts\tools\Migrations' -c execute -v 'oat\qtiItemPci\migrations\Version202103181527241465_qtiItemPci'
```

**Note:** due to some bad design in the PCI helpers, we need MathJax to be installed for the unit test to complete

How to test:
- run the unit tests (`cd tao/views/build && npx grunt connect:dev qtiitempcitest`)
- see the main PR on [taoOutcomeUI](https://github.com/oat-sa/extension-tao-outcomeui/pull/374) for a more complete test case
